src/ApplenewsManager.php

```diff
@@ -190,7 +190,13 @@ public static function getArticle(EntityInterface $entity, $field_name) {
         ->condition('entity_id', $entity->id())
         ->condition('field_name', $field_name)
         ->execute();
-      return \Drupal::entityTypeManager()->getStorage('applenews_article')->load(current($ids));
+      if (!empty($ids)) {
+        $articles = \Drupal::entityTypeManager()->getStorage('applenews_article')->loadMultiple($ids);
+        // We expect to have only one article.
+        foreach ($articles as $article) {
```

Comment:
Looks like `return array_shift($articles)` would be simpler, but not a blocker.

Reply:
yeah, I looked for some way to retrieve just one item by `load` instead of `loadMultiple` and there is no easy way for that. so went with the loop, as it's `loadMultiple`.